sos/plugins/ovirt_provider_ovn.py

```diff
@@ -25,9 +25,10 @@ class OvirtProviderOvn(Plugin, RedHatPlugin):
     profiles = ('virt',)
 
     def setup(self):
-        self.add_copy_spec(['/etc/ovirt-provider-ovn/ovirt-provider-ovn.conf'])
+        self.add_copy_spec('/etc/ovirt-provider-ovn/ovirt-provider-ovn.conf')
 
+        self.limit = self.get_option('log_size')
```

Comment:
There's no need to make this a member of the `OvirtProviderOvn` class: it is assigned and referenced in only a single method.

The pattern in the `openvswitch` plugin is perfectly correct.